recipe/conda_build_config.yaml

```diff
@@ -0,0 +1,3 @@
+MACOSX_SDK_VERSION:        # [osx and x86_64]
+  - "10.13"                # [osx and x86_64]
+
```

Comment:
This should not be necessary - in any case, it would only be active after rerendering. Can you remove that file entirely for the time being?

Reply:
Hmmmm... Seems that `std::aligned_alloc` is [actually](https://en.cppreference.com/w/c/memory/aligned_alloc) a function from the C standard library, not from the C++ one. So it's possible that we'll have to bump the `MACOSX_SDK_VERSION` _and_ `MACOSX_DEPLOYMENT_TARGET` after all, plus a rerender.